recipes/libvault/all/conanfile.py

```diff
@@ -42,8 +37,8 @@ def configure(self):
             del self.options.fPIC
```

Comment:
```suggestion
            del self.options.rm_safe("fPIC")
```

On Conan 2.x, once removed by config_options, it will fail to remove again here.

Reply:
Thanks, I only noticed the rm_safe in documentation yesterday and was wondering if I should use it.

Is the convention to del in addition to rm_safe?